lib/active_shipping/shipment_event.rb

```diff
@@ -2,14 +2,18 @@ module ActiveShipping
   class ShipmentEvent
     attr_reader :name, :time, :location, :message, :type_code
 
-    def initialize(name, time, location, message = nil, type_code = nil)
-      @name, @time, @location, @message, @type_code = name, time, location, message, type_code
+    def initialize(name, time, location, message = nil, type_code = nil, timezone_ambiguous = true)
```

Comment:
Should this still default to true? I think we assume it is _not_ ambiguous by default right? For carriers that do not set this. We don't really test that path.